src/main/java/org/prebid/server/util/HttpUtil.java

```diff
@@ -129,7 +129,7 @@ public static MultiMap headers() {
      */
     public static void addHeaderIfValueIsNotEmpty(MultiMap headers, CharSequence headerName, CharSequence headerValue) {
         if (StringUtils.isNotEmpty(headerValue)) {
-            headers.add(headerName, headerValue);
+            headers.set(headerName, headerValue);
```

Comment:
There are headers that can have multiple entries, if we will talk about XFF, it's also a header that can contain multiple values.
But if it's related bidder code, and u want only one value, pls do it in bidder

Reply:
Ok, then maybe i will add **setHeaderIfValueIsNotEmpty** utility method?